drivers/soundwire/intel.h

```diff
@@ -47,15 +50,100 @@ struct sdw_intel {
 #endif
 };
 
-int intel_link_startup(struct auxiliary_device *auxdev);
-int intel_link_process_wakeen_event(struct auxiliary_device *auxdev);
+#define cdns_to_intel(_cdns) container_of(_cdns, struct sdw_intel, cdns)
 
-struct sdw_intel_link_dev {
-	struct auxiliary_device auxdev;
-	struct sdw_intel_link_res link_res;
-};
+#define INTEL_MASTER_RESET_ITERATIONS	10
+
+static inline void sdw_intel_debugfs_init(struct sdw_intel *sdw)
+{
+	if (sdw && sdw->link_res && sdw->link_res->hw_ops &&
+	    sdw->link_res->hw_ops->debugfs_init)
+		sdw->link_res->hw_ops->debugfs_init(sdw);
+}
+
+static inline void sdw_intel_debugfs_exit(struct sdw_intel *sdw)
+{
+	if (sdw && sdw->link_res && sdw->link_res->hw_ops &&
+	    sdw->link_res->hw_ops->debugfs_exit)
+		sdw->link_res->hw_ops->debugfs_exit(sdw);
+}
+
+static inline int sdw_intel_register_dai(struct sdw_intel *sdw)
+{
+	if (sdw && sdw->link_res &&  sdw->link_res->hw_ops &&
```

Comment:
one more whitespace before sdw->link_res->hw_ops.

Reply:
> one more whitespace before sdw->link_res->hw_ops.

In fact, the double whitespace is seen in every `if (sdw && sdw->link_res &&  sdw->link_res->hw_ops &&`. ;)

Reply:
fixed with macros